### DISCUSSION TITLE
NocoDB as a frame (iFrame) [Aug 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6163] [Discord Threads]
> 🙋‍♂️ **vishnuv**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### For the view that you wish to include in your application,    
#### > 1. Open view   
#### > 2. Click on "Share" button on top right menu bar   
#### > 3. Under "Shared base link"   
#### > 3a. Select 'Anyone with the link'   
#### > 4. Base is shared now. You can copy iFrame code to be embedded into your application by clicking on </> icon    
#### > 5. Insert this into your HTML code  
#  
> 🙋‍♂️ **pradeep_lotus**  
#### I want to open whole application in Iframe plus I want to capture a param passed from Iframe  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/944459485949349898)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
